jquery/confirmable.d.ts

```diff
@@ -69,31 +79,35 @@ interface Options {
 interface I18N {
     /**
      * Text to use for the confirmation question.
+     * Defaults to `Are you sure?`.
      */
     confirm: string;
 
     /**
      * Text to use for the 'No' button.
+     * Defaults to `No`.
      */
     no: string;
 
     /**
-     * Title text to use for the 'No' button.
+     * Optional title text to use for the 'No' button.
      */
     noTitle: string | undefined;
```

Comment:
should this be `noTitle?: string` instead?
```suggestion
    noTitle?: string;
```

Reply:
Sorry for the late response. Yeah, that's a mistake, thank you very much, this is applied in the latest commit. `defaultOptions` has all its properties required (and set to `undefined` if missing), so I suggest either we keep it as a special case, or we use `noTitle?: string | undefined;` to match both use cases.